google/cloud/ndb/model.py

```diff
@@ -3861,6 +3899,48 @@ def _get_value_size(self, entity):
             values = [values]
         return len(values)
 
+    def _to_datastore(self, entity, data, prefix="", repeated=False):
+        """Override of :method:`StructuredProperty._to_datastore`.
```

Comment:
Rather, an override of `Property._to_datastore`, no?

Reply:
Yep.